pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -38,8 +40,8 @@ func testStableSystemOperatorStateTransitions(events monitorapi.Intervals) []*ju
 	return testOperatorStateTransitions(events, []configv1.ClusterStatusConditionType{configv1.OperatorAvailable, configv1.OperatorDegraded}, except)
 }
 
-func testUpgradeOperatorStateTransitions(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
-	except := func(operator string, condition *configv1.ClusterOperatorStatusCondition) (string, error) {
+func testUpgradeOperatorStateTransitions(events monitorapi.Intervals, clientConfig *rest.Config) []*junitapi.JUnitTestCase {
+	except := func(operator string, condition *configv1.ClusterOperatorStatusCondition, clientConfig ...*rest.Config) (string, error) {
```

Comment:
I might be missing something here. Why do we need multiple clientConfig here?

Reply:
I had defined that as a variadic function to minimise the code changes required in testOperatorStateTransitions code, have changed it to another approach now, please check